Packs/Intezer/Integrations/IntezerV2/IntezerV2.yml

```diff
@@ -3,157 +3,260 @@ commonfields:
+      deprecated: false
+      description: Checks file reputation of the given hash, supports SHA256, SHA1 and MD5
+      execution: false
+      outputs:
+        - contextPath: Intezer.Analysis.ID
+          description: Intezer analysis id
+          type: string
+        - contextPath: Intezer.Analysis.Status
+          description: status of the analysis
+          type: string
+        - contextPath: Intezer.Analysis.Type
+          description: type of the analysis
+          type: string
+    - name: intezer-get-latest-report
+      arguments:
+        - default: false
+          description: Hash of the file to query. Supports SHA256, MD5 and SHA1
+          isArray: false
+          name: file_hash
+          required: true
+          secret: false
+      deprecated: false
+      description: Checks file reputation of the given hash, supports SHA256, SHA1 and MD5 by looking at the latest available report
+      execution: false
+      outputs:
+        - contextPath: File.SHA256
+          description: Hash SHA256
+          type: string
+        - contextPath: File.Malicious.Vendor
+          description: For malicious files, the vendor that made the decision
+          type: string
+        - contextPath: DBotScore.Indicator
+          description: The indicator we tested
+          type: string
+        - contextPath: DBotScore.Type
+          description: The type of the indicator
+          type: string
+        - contextPath: DBotScore.Vendor
+          description: Vendor used to calculate the score
+          type: string
+        - contextPath: DBotScore.Score
+          description: The actual score
+          type: number
+        - contextPath: File.Metadata
+          description: Metadata returned from Intezer analysis (analysis id, analysis url, family, family type, sha256, verdict, sub_verdict). Metedata will be retuned only for supported files.
+          type: Unknown
+        - contextPath: File.ExistsInIntezer
+          description: Does the file exists on intezer genome database
+          type: Boolean
+    - name: intezer-analyze-by-file
+      arguments:
+        - default: true
+          description: The file entry id to upload
+          isArray: false
+          name: file_entry_id
+          required: true
+          secret: false
+      deprecated: false
+      description: Checks file reputation for uploaded file (up to 150MB)
+      execution: false
+      outputs:
+        - contextPath: Intezer.Analysis.ID
+          description: Intezer analysis id
+          type: string
+        - contextPath: Intezer.Analysis.Status
+          description: status of the analysis
+          type: string
+        - contextPath: Intezer.Analysis.Type
+          description: type of the analysis
+          type: string
+    - name: intezer-get-analysis-result
+      arguments:
+        - default: false
+          description: The analysis ID we want to get results for
+          isArray: false
```

Comment:
Why the change in this command?
It will break backward compatibility. 
The playbooks are using generic polling which uses this command as it, and this change will break it.

Reply:
i see.. so i have to be able to handle arrays, do i have to return different CommandResult for each analysis id or just 1 with entry in the context for each? @yaakovpraisler

Reply:
ok nevermind, i return a list of command results for each analysis :)

Reply:
If the argument continues to support comma-separated values, so you can also leave the outputs as they were until now.

Reply:
it's not works as an array so it's good :+1: